client/packages/inventory/src/Stocktake/DetailView/SidePanel.tsx

```diff
@@ -88,6 +88,7 @@ export const SidePanel = () => {
     selectedRows: [stocktake],
     deleteAction,
     messages: {
+      // TODO differentiate betweeen duplicate stocktakeNumber on delete where stores have been merged (https://github.com/msupply-foundation/open-msupply/pull/6789)
       confirmMessage: t('messages.confirm-delete-stocktake', {
         number: stocktake?.stocktakeNumber,
```

Comment:
Niche case where merged stores have two stocktakes of the same number, and delete message is ambiguous. I considered removing the stocktake number from this message but figure it is a rare case where it would actually cause confusion, and would be helpful in most cases.